ql/ql/test/queries/performance/MissingNoInline/MissingNoInline.expected

```diff
@@ -0,0 +1 @@
+| Test.qll:8:11:8:25 | ClasslessPredicate missingNoInline | This predicate might be inlined. |
```

Comment:
```suggestion
| Test.qll:8:11:8:25 | ClasslessPredicate missingNoInline | This predicate should probably be marked pragma[noinline] to prevent inlining. |
```